### ISSUE TITLE
Initial check and reload() don't work

### ISSUE THREAD

**Issue by iamcxds** [barreiroleo/ltex_extra.nvim#53]
**Describe the bug**
The initial check and `reload()` doesn't work properly. It can't load my dictionary.
The log has only
`[ltex_extra] [TRACE 16:10:07] ~/.local/share/nvim/lazy/ltex-extra.nvim/lua/lt
ex_extra/commands-lsp.lua:79: updateConfigFull
`
then nothing happens, no error or further step.

But it works properly if I make a code action of adding new words. 

**To Reproduce**
1. Your settings for ltex are: 
```lua
lspconfig.ltex.setup {
  capabilities = capabilities,
  on_attach = function(client, bufnr)
    on_attach(client, bufnr) -- rest of your on_attach process.
    require("ltex_extra").setup {
      load_lang={"en-US",},
      init_check = true,
      path =  "~/.local/share/ltex",
      log_level="trace",
    }
  end,
}
```

**System:**
 - OS: Ubuntu
 - Neovim version: 0.9
 - LTeX server intallation method: Mason

**Additional context**
I used Nvchad to setup .

**Comment by barreiroleo** [comment]
Can you try setting the path as `path = vim.fn.expand("~/.local/share/ltex")`?

**Comment by iamcxds** [comment]
> Can you try setting the path as `path = vim.fn.expand("~/.local/share/ltex")`?

This doesn't fix.

**Comment by iamcxds** [comment]
It seems like some problem with getting access to the options, 
If I manually run `reload()` it stops very quickly, but `reload({"en-US")` works, 
from the code the problem seems on calling `package.loaded.ltex_extra.opts.load_langs`

**Comment by barreiroleo** [comment]
I was able to reproduce it and you have a typo in the settings. It's `load_langs` not `load_lang`.
In the other hand, I need to rewrite this thing.

**Comment by michael-pfister** [comment]
Not sure as to why this bug was closed on an open source project. I am experiencing the same problem. Even if OC rewrote the entire plugin, someone could see this issue and make a PR for the old one.

@iamcxds did you find a solution for this? I also noticed that it'll start working again if we reload multiple languages `:lua require('ltex_extra').reload({'en-US', 'de-DE'})`.

I tried doing this automatically after the plugin was initialized but it didn't seem to work.
